Source/gui/SNTAppDelegate.m

```diff
@@ -125,4 +134,31 @@ - (void)setupMenu {
   [NSApp setMainMenu:mainMenu];
 }
 
+#pragma mark Push Notifications
+
+- (void)application:(NSApplication *)application
+  didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)tokenData {
```

Comment:
```suggestion
    didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)tokenData {
```

Reply:
Unless clang-format disagrees, which would mean it's not following the Google style guide

Reply:
:/ yes, clang-format is doing this

Reply:
Ah I see we have misconfigured clang-format. This is fine for now, I'll fix it alter.